src/main/java/uk/gov/justice/digital/hmpps/cmd/api/client/CsrClient.kt

```diff
@@ -76,15 +76,15 @@ class CsrClient(
   }
 
   private fun getDetails(from: LocalDate, to: LocalDate, region: String): Collection<CsrDetailDto> {
-    log.debug("User Details: finding User ${authenticationFacade.currentUsername}, Region $region")
+    log.debug("User Details: finding User ${authenticationFacade.username}, Region $region")
```

Comment:
technically this was previously calling `auth.principal` but since we know it was a user that is always passed through from the auth token rather than a `clientId` we can change to `username` instead.